src/generate/register.rs

```diff
@@ -216,26 +213,19 @@ pub fn fields(
     let can_write = access != Access::ReadOnly;
 
     // TODO enumeratedValues
+    let inline = quote! { #[inline(always)] };
```

Comment:
Have you checked whether this is worth the harder to read quote sections below? If we do something like this we ideally wouldn't repeat the quoting for each register we process but do it exactly once and then reuse it.